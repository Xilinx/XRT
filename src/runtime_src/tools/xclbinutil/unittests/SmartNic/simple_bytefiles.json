{
    "smartnic": {
        "schema_version": {
            "major": "5",
            "minor": "1",
            "patch": "2"
        },
        "extensions": [
            {
                "setup": {
                    "ebpf_file": "dummy_bytefile.txt"
                },
                "cam_instances": [
                    {
                        "name": "foo_my_cam_instance_1",
                        "id": "0",
                        "config_file": "dummy_bytefile.txt",
                        "driver_index": "0"
                    }
                ]
            }
        ]
    }
}
