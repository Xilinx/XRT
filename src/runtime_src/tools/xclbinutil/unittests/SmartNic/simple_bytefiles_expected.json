{
    "smartnic": {
        "schema_version": {
            "major": "5",
            "minor": "1",
            "patch": "2"
        },
        "extensions": [
            {
                "setup": {
                    "ebpf": "44756D6D7920427974652046696C650A"
                },
                "cam_instances": [
                    {
                        "name": "foo_my_cam_instance_1",
                        "id": "0",
                        "config": "44756D6D7920427974652046696C650A",
                        "driver_index": "0"
                    }
                ]
            }
        ]
    }
}
